Plugins/FastScaling/ImageResizer.Plugins.FastScaling.h

```diff
@@ -21,17 +21,17 @@ namespace ImageResizer{
 
 #undef FASTSCALING_ENUMS_MANAGED
 #undef STATUS_CODE_NAME
-#undef FLOATSSPACE_NAME
+#undef FLOATSPACE_NAME
```

Comment:
The `undef` guard worries me.

That said, I'm moderately confident this change is correct.

Reply:
All instances are currently consistent and can be changed.